README.md

```diff
@@ -44,12 +44,51 @@ Enter your DSN in the NuxtJS config file. Additional config settings can be foun
 
 In a Vue component, `Sentry` is available as `this.$sentry`, so we can call functions like
 
-```
+``` js
 this.$sentry.captureException(new Error('example'))
 ```
 
 where `this` is a Vue instance.
 
+### Usage in the `fetch` and `asyncData` methods
+
+While using nuxt's `fetch` and `asyncData` methods, the `$sentry` object is available in the root Vue object `app` like other nuxt plugins:
+
+``` js
+async fetch ({ app, params, store }) {
+  try {
+    let { data } = await axios.get(`https://my-api/posts/meta/${params.id}`)
+    store.commit('setMeta', data)
+  } catch (error) {
+    app.$sentry.captureException(new Error(error))
+  }
+}
+
+async asyncData ({ app, params }) {
+  try {
+    let { data } = await axios.get(`https://my-api/posts/${params.id}`)
+    return { title: data.title }
+  } catch (error) {
+    app.$sentry.captureException(new Error(error))
```

Comment:
Still not passing `error` in all examples. :)

Reply:
My bad!